authz/providers/azure/rbac/rbac.go

```diff
@@ -374,6 +375,10 @@ func (a *AccessInfo) CheckAccess(request *authzv1.SubjectAccessReviewSpec) (*aut
 }
 
 func (a *AccessInfo) sendCheckAccessRequest(ctx context.Context, checkAccessURL url.URL, checkAccessBody *CheckAccessRequest, ch chan *authzv1.SubjectAccessReviewStatus) error {
+	if c, ok := ctx.Value(oauth2.HTTPClient).(*http.Client); ok {
+		a.client = c
```

Comment:
Same comment here, don't modify `a.client`, store it in a local variable.

Reply:
fixed. Could you please check whether the fix meets the requirement? Thanks